trackingC_setup.Rmd

```diff
@@ -225,7 +233,8 @@ run_hector <- function(pdata, c) {
     message("\tSetting", appendLF = FALSE)
   for(p in colnames(pdata)) {
     message(" ", p, appendLF = FALSE)
-    setvar(c, NA, do.call(p, list()), pdata[p][[1]], units_vector[p])
+    if(p == scalar) setvar(c, core$strtdate:core$enddate, LUC_EMISSIONS(), (luc$value * pdata["LUC_SCALAR"][[1]]), "Pg C/yr")
```

Comment:
A comment to explain what's going on here might be useful.

Also, instead of explicitly using the string "Pg C/yr", why not use the `name_vector` and `units_vector` mechanism we already have? That seems clearer.

Reply:
Instead of `core$strtdate:core$enddate` might do `luc$year`.

Reply:
For using `units_vector`, the scalar itself is unitless.

Reply:
Ahhhh.....got it, thanks.